contrib/storage-jdbc/pom.xml

```diff
@@ -15,62 +15,241 @@
 -->
-<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <artifactId>drill-contrib-parent</artifactId>
-    <groupId>org.apache.drill.contrib</groupId>
-    <version>1.2.0-SNAPSHOT</version>
-  </parent>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>drill-contrib-parent</artifactId>
+        <groupId>org.apache.drill.contrib</groupId>
+        <version>1.2.0-SNAPSHOT</version>
+    </parent>
 
-  <artifactId>drill-jdbc-storage</artifactId>
+    <artifactId>drill-jdbc-storage</artifactId>
 
-  <name>contrib/jdbc-storage-plugin</name>
+    <name>contrib/jdbc-storage-plugin</name>
 
-  <dependencies>
+    <properties>
+        <mysql.connector.version>5.1.36</mysql.connector.version>
+        <derby.reserved.port>20000</derby.reserved.port>
+        <mysql.reserved.port>20001</mysql.reserved.port>
+        <derby.database.name>drill_derby_test</derby.database.name>
+        <mysql.database.name>drill_mysql_test</mysql.database.name>
+    </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.drill.exec</groupId>
+            <artifactId>drill-java-exec</artifactId>
+            <version>${project.version}</version>
+        </dependency>
 
-    <dependency>
-      <groupId>org.apache.drill.exec</groupId>
-      <artifactId>drill-java-exec</artifactId>
-      <version>${project.version}</version>
-    </dependency>
-    
-    <!-- Test dependencies -->
-    <dependency>
-      <groupId>org.apache.drill.exec</groupId>
-      <artifactId>drill-java-exec</artifactId>
-      <classifier>tests</classifier>
-      <version>${project.version}</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.drill</groupId>
-      <artifactId>drill-common</artifactId>
-      <classifier>tests</classifier>
-      <version>${project.version}</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>com.yammer.metrics</groupId>
-      <artifactId>metrics-core</artifactId>
-      <version>2.1.1</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.derby</groupId>
-      <artifactId>derbyclient</artifactId>
-      <version>10.11.1.1</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.derby</groupId>
-      <artifactId>derbynet</artifactId>
-      <version>10.11.1.1</version>
-      <scope>test</scope>
-    </dependency>    
-  </dependencies>
+        <!-- Test dependencies -->
+        <dependency>
```

Comment:
It looks like you moved from 2 space to four space indenting here. Should probably fix. Appears to have happened above as well. Can you check your idea for 2 space/120 character width?